cypress/unit/store-getter-getCategoricalJsonOutput.cy.js

```diff
@@ -0,0 +1,106 @@
+import { getters } from "~/store";
+
+const category = "category1";
+const columnName = "column1";
```

Comment:
:cherries:: why do we need these variables? Don't they just refer to what's mocked? Is the idea that I could change the name of the mocked stuff and then I can change it all in one place? Not sure this is worth the reduced readability.

Reply:
> why do we need these variables? 

These are left over from Jonathan's code.

>  Is the idea that I could change the name of the mocked stuff and then I can change it all in one place? 

I think so

I'll clean them up.